src/components/Farmhand/Farmhand.js

```diff
@@ -390,10 +419,16 @@ export default class Farmhand extends Component {
       cowInventory: [],
       cowsSold: {},
       cowsTraded: 0,
+      cowTradeTimeoutId: -1,
       cropsHarvested: {},
       dayCount: 0,
       farmName: 'Unnamed',
       field: createNewField(),
+      fieldMode: OBSERVE,
+      getCowAccept: () => {},
```

Comment:
super minor suggestion, but for readability i like to create a `const noop = () => {}` and then use that instead of creating all these empty functions as default values. i sometimes go as far as to make a `noop` util so it can be imported and used across a codebase, but i'm not strongly opinionated on doing that

Reply:
I think that's a great call-out. A global `noop` util is long overdue. Added in https://github.com/jeremyckahn/farmhand/commit/6397358d4cff7e87f9827e49d9fe02cfabc6abe8!